### DISCUSSION TITLE
Can Nuxt Content be used to handle markdown content coming from an API ?

### DISCUSSION THREAD

**Discussion by OptiZonion** [nuxt/content#1650] [Q&A]
Hello,

I'm trying to setup a project that will process API Documentations in markdown to display them to the user. I'm using Nuxt and after fiddling with existing solutions (AsciiDoctorJS, VuePress, etc), I encountered Nuxt Content which might match my needs.

The issue is that all of the solutions I've found, including Nuxt Content, always rely on local static files : MD files are stored in a fixed path that is being monitored by the solution. However, in my case, the MD files are stored in a database that is being queried using a REST API. In that configuration, I can't really make it work as intended. I've encountered beginning of solutions on the internet but nothing too conclusive.

I'm no front-end or Nuxt expert (pretty much not an expert in anything really) but what I want to know if is it feasible and if so, how to achieve, conceptually ? Because whatever I try with the asyncData fetching method, it's not working as soon as I try to load a content that is not local.

Thanks !

**Comment by niklasfjeldberg** [comment]
I would also like to know this, did you ever find out @DannyBiker?

**Comment by ManasMadrecha** [comment]
New V2 supports multi sources, and not just local content folder.

**Reply by lavellotron** [reply]
Can you say more @ManasMadrecha? As in, what portion of the Nuxt Content API would we use? I'm loading Markdown from Storyblok for my pages and want to know if I can use Nuxt Content for rendering. Thanks in advance for any tips!

**Reply by ManasMadrecha** [reply]
You can see the Nuxt Content docs, wherein in the Configuration page, you can see that you can have multiple `sources`. These sources can be your local folder or git repositories.

However, as you are already loading markdown from storyblok, you should use the Storyblok module, and not the Nuxt Content module.

Check this: https://nuxt.com/modules/storyblok

**Reply by mklueh** [reply]
Would it be possible to allow functions as a source? I'd love to connect Directus CMS to it
